libs/libgmime/Makefile

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libgmime
+PKG_VERSION:=2.6.23
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://git.gnome.org/browse/gmime/snapshot/
```

Comment:
This should use @GNOME (https://download.gnome.org/sources/gmime/2.6/)
https://github.com/lede-project/source/blob/master/scripts/download.pl#L243